test/TrusonaTest.js

```diff
@@ -188,46 +188,45 @@ describe('Trusona', () => {
     })
 
     context('with missing identifier', () => {
-      it('should throw a ValidationError', async () => {
+      it('should throw a TrusonaError', async () => {
```

Comment:
The following tests were failing because the service is not returning a `ValidationError` anymore, now it's returning a `TrusonaError`.

Reply:
@JUANDIEGOGARITA rerun this tests. I fixed something in the `trusona-service` that may have caused this.

Reply:
@thitu I just did, but these tests are still working with `TrusonaError` 🤷‍♂

Reply:
then merge away!